libs/classifiers/classifier.py

```diff
@@ -18,7 +18,8 @@ def inference(self, objects):
         return self.classifier.inference(objects)
 
     def object_post_process(self, object, classifier_result, classifier_score):
-        if object['face'] is not None and classifier_score > self.min_threshold:
-            object['face_label'] = classifier_result
-        else:
-            object['face_label'] = -1
+        if object.get('face') is not None:
```

Comment:
Again, if you intend to check existence, you have to check with `in` operator, not `get`. Explained in your other PR for yolov3. 

Thanks for your bughunt, great work.

Reply:
`get` and `in` both work. I'll change it to in